tachyon/zk/plonk/halo2/prover.h

```diff
@@ -158,6 +158,15 @@ class Prover : public ProverBase<PCS> {
             LookupProver::GetNumPermutedPairsCommitments(lookup_provers));
       }
       LookupProver::BatchCommitPermutedPairs(lookup_provers, this, commit_idx);
+    } else if constexpr (LS::type == lookup::Type::kLogUp) {
+      LookupProver::BatchCompressPairs(lookup_provers, domain, cs.lookups(),
+                                       theta, column_tables);
+      LookupProver::BatchComputeMPolys(lookup_provers, this);
+      if constexpr (PCS::kSupportsBatchMode) {
```

Comment:
Nitpicky, but can you add an empty line above to match the previous if statement?